### PR TITLE
Commit by market

### PR DESCRIPTION
**Pull request by martineehrlich**
## Description
We want to commit a balance in a currency to a specific market so that we don't have to hard code BTC -> LTC. This PR changes the commit-balance command, to commit, and now makes the user specify a market.

## Todos
- [x] Tests
- [x] Documentation
- [x] Link to Trello

### CODE REVIEW

**Comment by martineehrlich** [comment]
[Commit by market](https://trello.com/c/Xk8C3dKO/605-commit-by-market)

File: broker-cli/commands/wallet.js [hunk 1]
```diff
@@ -109,21 +109,22 @@ async function newDepositAddress (args, opts, logger) {
...
  * @param {Object} args.symbol
  * @param {Object} opts
  * @param {String} [opts.rpcAddress] broker rpc address
+ * @param {String} [opts.market] market to commit funds to
  * @param {Logger} logger
  * @return {Void}
  */
-async function commitBalance (args, opts, logger) {
+async function commit (args, opts, logger) {
   const { symbol, amount } = args
```
Comment (dannypaz) [inline broker-cli/commands/wallet.js]: We no longer need symbol for commit.

File: broker-cli/commands/wallet.js [hunk 2]
```diff
@@ -344,8 +345,9 @@ module.exports = (program) => {
 
           args.symbol = symbol
```
Comment (dannypaz) [inline broker-cli/commands/wallet.js]: We can remove symbol as its not needed for commit

File: broker-cli/commands/wallet.js [hunk 3]
```diff
@@ -367,9 +369,10 @@ module.exports = (program) => {
     .command('wallet balance', 'Current daemon wallet balance')
     .command('wallet new-deposit-address', 'Generates a new wallet address for a daemon instance')
     .argument('<symbol>', `Supported currencies for the exchange: ${SUPPORTED_SYMBOLS.join('/')}`)
-    .command('wallet commit-balance')
+    .command('wallet commit')
     .argument('<symbol>', `Supported currencies for the exchange: ${SUPPORTED_SYMBOLS.join('/')}`)
     .argument('[amount]', 'Amount of currency to commit to the relayer', validations.isDecimal)
+    .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
```
Comment (dannypaz) [inline broker-cli/commands/wallet.js]: Might be good to have available market names in the description

**Comment by dannypaz** [comment]
@martineehrlich we'll need to make sure all documentation (docs repo and sparkswap.com) are updated with these changes

File: broker-cli/commands/wallet.js (hunk 3 above)
Comment (martineehrlich) [reply broker-cli/commands/wallet.js]: Will do that here: https://trello.com/c/a5xtnFbv/655-get-supported-markets-ccxt

**Pull request merged**